src/Tizen.NUI/src/public/Xaml/XamlParseException.cs

```diff
@@ -27,6 +27,23 @@ static private StringBuilder GetStackInfo()
             return ret;
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Please add brief description of the API when it is public (or hidden).

This is the example,
```C#
/// <summary> Initializes a new instance. </summary>
[EditorBrowsable(EditorBrowsableState.Never)]
public XamlParseException()
{
}

```

Reply:
I see